content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -16,6 +16,7 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 1. [Kubernetes platform setup](/docs/setup/kubernetes/platform-setup/):
   * [Minikube](/docs/setup/kubernetes/platform-setup/minikube/)
   * [Google Container Engine (GKE)](/docs/setup/kubernetes/platform-setup/gke/)
+  * [Oracle Cloud Infrastructure (OKE)](/docs/setup/kubernetes/platform-setup/oci/)
```

Comment:
Would you mind alphabetically sorting this list?

Reply:
I don't mind at all. This ends up mixing in non-cloud providers with cloud providers. The list renders on docs/setup/kubernetes/platform-setup in a way that groups the cloud providers together, which I could also do. For now, it's just alphabetized, let me know which you'd prefer.